textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/queryrewriter/FuzzyTokenizer.java

```diff
@@ -43,36 +44,106 @@ public FuzzyTokenizer(String phrase) {
     }
 
     /**
+     * When called returns a single-member list containing most-likely tokenization of phrase
```

Comment:
It does make sense for `fuzzyTokenizer.getFuzzyTokens()` to be a static function. We are making changes as suggested.